channels/views.py

```diff
@@ -80,10 +84,29 @@ class CommentListView(ListCreateAPIView):
     """
     serializer_class = CommentSerializer
 
+    def get_serializer_context(self):
+        context = {
```

Comment:
Can you get this from the super class instead: `context = super().get_serializer_context()`

Reply:
oh good idea